doc/man3/UI_new.pod

```diff
@@ -156,16 +156,16 @@ string and may include encodings that will be processed by the other
 method functions.
 
 UI_add_user_data() adds a user data pointer for the method to use at any
-time.  The builtin UI method doesn't care about this info.  Note that several
-calls to this function doesn't add data, it replaces the previous blob
+time.  The builtin UI method does not care about this info.  Note that several
+calls to this function does not add data, it replaces the previous blob
```

Comment:
Since we're [sic] here already, how about "Note that repeated calls to this function do not append; each call replaces the previous blob with the one given as an argument."?